packages/api/tests/endpoints/test-granules.js

```diff
@@ -597,6 +597,28 @@ test('DELETE deleting an existing unpublished granule', async (t) => {
   ]);
 });
 
+test('DELETE for a granule with a file not present in S3 succeeds', async (t) => {
```

Comment:
This just verifies that, even if we don't check for the existence of the file before deleting it, it still works.

Reply:
Fair enough.  We're trusting localstack's implementation here though.   🤔